index.js

```diff
@@ -10,6 +10,8 @@ YellowBox.ignoreWarnings([
   // TODO this a warning from react-navigation, see: https://reactnavigation.org/docs/en/common-mistakes.html
   // This happens in our app because the package-stack architecture we have for native apps
   'You should only render one navigator',
+  'Warning: componentWillReceiveProps is deprecated',
```

Comment:
Is that necessary? Maybe getting a reminder could be useful in case we forget that these warnings were suppressed and some weird bug happens in the future. WDYT?

Reply:
We don’t have this in our codebase, this comes from 3rd party libraries, and is just noise 😊